src/System.Private.ServiceModel/pkg/System.Private.ServiceModel.pkgproj

```diff
@@ -2,15 +2,11 @@
 <Project ToolsVersion="12.0" DefaultTargets="Build" xmlns="http://schemas.microsoft.com/developer/msbuild/2003">
   <Import Project="$([MSBuild]::GetDirectoryNameOfFileAbove($(MSBuildThisFileDirectory), dir.props))\dir.props" />
 
-  <PropertyGroup>
-    <PreventImplementationReference>true</PreventImplementationReference>
-  </PropertyGroup>
-
   <ItemGroup>
-    <ProjectReference Include="..\src\System.Private.ServiceModel.builds">
-      <AdditionalProperties>FilterToOSGroup=Windows_NT</AdditionalProperties>
-      <UndefineProperties>%(ProjectReference.UndefinedProperties);PackageTargetFramework</UndefineProperties>
-    </ProjectReference>
+    <ProjectReference Include="..\src\System.Private.ServiceModel.csproj" />
```

Comment:
I had reviewed that example but I wasn't sure if the entire content applied to this case. I didn't understand what the purpose was for the PLaceHolder files, I think I do now. In the updated PR I have included everything from that file except...
<SkipGenerationCheck>true</SkipGenerationCheck>

As it doesn't seem to be needed.